website/docs/Examples/Integrate - OpenAI.md

```diff
@@ -0,0 +1,184 @@
+FLAML has integrated the OpenAI's completion API. In this example, we will tune several hyperparameters including the temperature, prompt and n to optimize the inference performance of OpenAI's completion API for a code generation task. Our study shows that tuning hyperparameters can significantly affect the utility of the OpenAI API. For more details please refer to our paper: http://arxiv.org/
+### Prerequisites
+
+Install the [openai] option. You should install version 1.1.3, this version is subject to change from time to time.
+```bash
+pip install "flaml[openai]==1.1.3"
+```
+
+
+Setup your OpenAI key:
+```python
+import os
+
+if "OPENAI_API_KEY" not in os.environ:
+    os.environ["OPENAI_API_KEY"] = "<your OpenAI API key here>"
+```
+
```

Comment:
"For Azure OpenAI, please ..."
The notebook has information about it. Please add that information here too. Be clear that we support both OpenAI API and Azure OpenAI API.